wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/CheLanguageClient.java

```diff
@@ -84,4 +89,17 @@ public void logMessage(MessageParams message) {
         break;
     }
   }
+
+  @Override
+  public CompletableFuture<Void> registerCapability(RegistrationParams params) {
```

Comment:
I don't think it's a good idea to pretend we support dynamic registration when we really don't. Language servers that would correctly report their capabilities upon startup, might actually decide to use dynamic registration instead. In particular, this might break jdt.ls

Reply:
There is a dedicated class `ClientCapabilitiesProvider` that defines if we support dynamic registration or not and according to it we do not support dynamic registration and don't pretend we do, so there is no reason for your concern. 

I assume this stub is added to avoid `UnsupportedOperationException ` for cases when language server (by mistake) ignores our definition of dynamic registration support and still tries to register capability.

Reply:
But the error message in the log is correct, we get it because the implementation of the language server is broken. How will we detect that a language server is broken in this respect? At least we would have to put a warning in the log.